terraform/modules/daos_server/main.tf

```diff
@@ -60,17 +60,12 @@ locals {
   server_startup_script = templatefile(
     "${path.module}/templates/daos_startup_script.tftpl",
     {
+      servers           = local.servers
       first_server      = local.first_server
       daos_ca_secret_id = local.daos_ca_secret_id
       allow_insecure    = local.allow_insecure
-    }
-  )
-
-  configure_daos_content = templatefile(
```

Comment:
I believe you still need a configure daos script. How are you going to start the daos daemon on clients?

Reply:
@lsitkiew please put back the basic configure daos script.  It is used to start Clients in some configs.

Reply:
I analyzed the scripts and this functionality was moved to startup script on client side: https://github.com/daos-stack/google-cloud-daos/blob/develop/terraform/modules/daos_client/templates/daos_startup_script.tftpl#L29-L30
Besides this, that file was not used by Terraform in any of the module, so that why I decided to deleted it. 
I didn't have any issue with not started DAOS agent.